src/esaml_cowboy.erl

```diff
@@ -74,33 +74,27 @@ reply_with_req(IDP, SignedXml, RelayState, Req) ->
         {response, esaml:logoutresp(), RelayState::binary(), Req} |
         {error, Reason :: term(), Req}.
 validate_logout(SP, Req) ->
-    {Method, Req} = cowboy_req:method(Req),
-    case Method of
+    case cowboy_req:method(Req) of
         <<"POST">> ->
             % XXX: compat hack, the cowboy_req:continue/1 function was introduced at the
             %      same time as the API change on body_qs/2, so we can use it to detect
             %      which argument layout we need to use. this way we can be compat with
             %      both cowboy <1.0 and 1.0.x.
-            {ok, PostVals, Req2} = case erlang:function_exported(cowboy_req, continue, 1) of
```

Comment:
Remove the comment above which applies to this compatibility code removed below!